middleware/logger_test.go

```diff
@@ -0,0 +1,43 @@
+package middleware
+
+import (
+	"errors"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/gofiber/fiber"
+	"github.com/gofiber/utils"
+	"github.com/valyala/bytebufferpool"
+)
+
+// go test -run Test_Middleware_Logger
+func Test_Middleware_Logger(t *testing.T) {
+	format := "${ip}-${ips}-${url}-${host}-${method}-${path}-${protocol}-${route}-${referer}-${ua}-${latency}-${status}-${body}-${error}-${bytesSent}-${bytesReceived}-${header:header}-${query:query}-${cookie:cookie}"
+	expect := "0.0.0.0--/test?query=query-example.com-GET-/test-http-/test-ref-ua-0s-500--error-5-0-header-query-cookie"
```

Comment:
you should take a regex here, the time of 0s for the latency is different in every test and therefore the test to my computer does not pass